get_prism_data.R

```diff
@@ -54,6 +54,7 @@ download_prism=function(){
 #########################################################
 check_if_prism_files_present=function(prism_ls, years){
   #Extract out all the variable names and dates
+  if (length(prism_ls$files) == 0){return(FALSE)}
```

Comment:
It's clear. But only because I know what all the variables are and where they came from. Someone looking thru this for the 1st time might have to do some unnecessary digging to figure out why a length check is needed and how that could happen.